cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -213,15 +214,24 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 		return errors.Wrap(err, "add command: failed to setup network")
 	}
 
+	containerInterfaceIndex := 1
 	ips := []*current.IPConfig{
 		{
-			Version: "4",
-			Address: *addr,
+			Version:   "4",
+			Address:   *addr,
+			Interface: &containerInterfaceIndex,
 		},
 	}
 
+	hostInterface := &current.Interface{Name: hostVethName}
+	containerInterface := &current.Interface{Name: args.IfName, Sandbox: args.ContainerID}
```

Comment:
let's do a dive deep to see whether this sandbox should be netns or containerID or can be either of them.

from cni spec:
```
sandbox: The isolation domain reference (e.g. path to network namespace) for the interface, or empty if on the host. For interfaces created inside the container, this should be the value passed via CNI_NETNS.
```

Reply:
As per the CNI spec it should be NS. Also PTP plugin I see they are passing netns. At least with Bandwidth plugin they check for sandbox == null to retrieve host interface - https://github.com/containernetworking/plugins/blob/8632ace977f4126c8ded601975f564571c66b922/plugins/meta/bandwidth/main.go#L156.

Reply:
Yeah, it should be netns - https://github.com/containernetworking/plugins/blob/e27c48b391539f5536918b9c379c59ef2793cb0d/plugins/sample/main.go#L81. netns should be empty for interfaces on hostns.

So, bandwidth plugin appears to be expecting the host interface as well in the result struct...